pack/df.json

```
@@ -1080,7 +1080,7 @@
         "quantity": 3,
         "side_code": "corp",
         "strength": 6,
-        "text": "When you rez this ice, take 1 bad publicity.\n[subroutine] Trash 1 installed Runner card.\n[subroutine]<trace>trace 6</trace>: If successful, the Runner cannot steal or trash Corp cards for the remainder of the run.",
+        "text": "When you rez this ice, take 1 bad publicity.\n[subroutine] Trash 1 installed Runner card.\n[subroutine] <trace>Trace 6</trace>: If successful, the Runner cannot steal or trash Corp cards for the remainder of the run.",
```

Looks like a colon snuck in too, so might as well get rid of that:

```suggestion
        "text": "When you rez this ice, take 1 bad publicity.\n[subroutine] Trash 1 installed Runner card.\n[subroutine] <trace>Trace 6</trace> If successful, the Runner cannot steal or trash Corp cards for the remainder of the run.",
```

---

GitHub doesn't want to let me commit this suggestion, did you want to include it @SpencerDub?

If not, I'll log a separate PR for it and just merge this one as-is.